scripts/lib.sh

```diff
@@ -374,62 +330,44 @@ get_extensions_from_dynatrace() {
-            echo "gcp:" >"$WORKING_DIR"/"$GCP_FUNCTION_NAME"/config/"${BASH_REMATCH[1]}".yaml
-            echo "$EXTENSION_GCP_CONFIG" >>"$WORKING_DIR"/"$GCP_FUNCTION_NAME"/config/"${BASH_REMATCH[1]}".yaml
-          fi
-        fi
-        ((AMOUNT_OF_EXTENSIONS_TO_UPLOAD+=1))
-        activate_extension_on_cluster "$EXTENSIONS_FROM_CLUSTER" "$EXTENSION_ZIP"
-        break
-      fi
-      echo -n "." | tee -a "$FULL_LOG_FILE"
-    done
-    rm extension.zip
-    rm "$EXTENSION_FILE_NAME".yaml
-    rm "$EXTENSION_ZIP"
-  done
-
-  if [[ "$AMOUNT_OF_EXTENSIONS_TO_UPLOAD" -eq "$AMOUNT_OF_NOT_UPLOADED_EXTENSIONS" ]]; then
-    err "Uploading all GCP extensions to Dynatrace failed. It can be a temporary problem with the cluster. Please run deployment script again in a while."
-    exit 1
-  fi
-
-  if [[ "$AMOUNT_OF_EXTENSIONS_TO_UPLOAD" -eq "$AMOUNT_OF_NOT_ACTIVATED_EXTENSIONS" ]]; then
-    err "Activating all GCP extensions on Dynatrace failed. It can be a temporary problem with the cluster. Please try activate your extensions on the cluster manually in a while."
+  GOOGLE_EXTENISION_ON_TENANT=$(echo "$LIST_OF_GOOGLE_EXTENSIONS" | jq -r '.totalCount')
+  
+  if [ "$GOOGLE_EXTENISION_ON_TENANT" -gt 0 ]; then
+      info ""
+      info "- there are alredy google extensions enabled on the tenant."
   else
-    AMOUNT_OF_ALL_FAILED_EXTENSIONS=$((AMOUNT_OF_NOT_UPLOADED_EXTENSIONS + AMOUNT_OF_NOT_ACTIVATED_EXTENSIONS))
-    if [[ "$AMOUNT_OF_EXTENSIONS_TO_UPLOAD" -eq "$AMOUNT_OF_ALL_FAILED_EXTENSIONS" ]]; then
-      err "Uploading and activating GCP extensions on Dynatrace failed.
-      It can be a temporary problem with the cluster. Please run deployment script again in a while.
-      For not activated but uploaded extensions - please try activate them on the cluster manually in a while."
-    fi
-  fi
+    info "- Activating default google extensions..."
+      DEFAULT_GOOGLE_EXTENSIONS=(
+        "com.dynatrace.extension.google-api"
+        "com.dynatrace.extension.google-app-engine"
+        "com.dynatrace.extension.google-bigquery"
+        "com.dynatrace.extension.google-cloud-functions"
+        "com.dynatrace.extension.google-cloud-storage"
+        "com.dynatrace.extension.google-compute-engine"
+        "com.dynatrace.extension.google-datastore"
+        "com.dynatrace.extension.google-filestore"
+        "com.dynatrace.extension.google-kubernetes-engine"
+        "com.dynatrace.extension.google-load-balancing"
+        "com.dynatrace.extension.google-pubsub"
+        "com.dynatrace.extension.google-pubsub-lite"
+        "com.dynatrace.extension.google-sql")
+
+    for GOOGLE_EXT in "${DEFAULT_GOOGLE_EXTENSIONS[@]}"
+      do
+        extensionName=$(echo "$GOOGLE_EXT" | tr -d '"')
+        info "Activating ${extensionName}..."
+        dt_api "/api/v2/hub/extensions2/${extensionName}/actions/addToEnvironment" "POST" ""
+      done
+
+    info ""
+    info "- extensions have been activated on the tenant."
```

Comment:
"default set of extensions" - 
Also here we need to add hint that you can browse the hub and search for gcp extensions and to enable more services